compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -34,10 +34,24 @@ def AMDAIECleanup :
       "mlir::iree_compiler::AMDAIE::createAMDAIECleanupPass()";
 }
 
+def AMDAIEAddLoweringStrategy :
+    Pass<"iree-amdaie-add-lowering-strategy", "mlir::iree_compiler::IREE::HAL::ExecutableVariantOp"> {
+  let summary = "Add lowering strategy configurations to be used";
+  let constructor = "mlir::iree_compiler::AMDAIE::createAMDAIEAddLoweringStrategyPass()";
+  let options = [
+    Option<"tilingStrategy", "tiling-strategy", "std::string", /*default=*/"",
```

Comment:
A `Stringef` option and string matching is better avoided. Can we just use a boolean option, and call it "usePackingStrategy" cause thats what we are using here.

Reply:
Ok, I will change to boolean. As for the naming, I think `usePackingStrategy` is a bit narrow as the unique feature here is the usage of a fixed/handcoded kernel/library function. So, I would prefer `useMicroKernelStrategy` or `useMLOpsLibStrategy` (MLOpsLib is the library of handcoded kernel functions). What do you think?

Reply:
Follow up this discussion, I think eventually we may have more tiling strategies. So maybe use an int option? And add explanations for what each value represent.

Reply:
@yzhang93 I agree that we likely need more tiling strategies so an int (or better enum?) would be great. @MaheshRavishankar mentioned yesterday that could be done in a follow-up PR? I made the change now to use a boolean.